src/utils/constants.ts

```diff
@@ -11,4 +11,17 @@ export const conditions = {
 
 export const signs = {
     CELSIUS: '\u00B0',
+    INTEREST: '\u0025',
+};
+
+export const caseOfCategories = {
```

Comment:
Not the best name, i get no info from it. You might use ExtraInfoType. The main thing is to combine everything you know from the task. So you know you are creating extra info component and the main difference between its instances is the type of its info, so why not name it as ExtraInfoType